<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="HR-47" failures="2" tests="2" name="Test12" time="10.753" errors="0" timestamp="2023-09-13T13:03:45 IST">
  <testcase classname="Day11.DataProviderDemo" name="testLogin" time="4.795">
    <failure type="java.lang.AssertionError" message="expected [nopCommerce demo store. Login] but found [nopCommerce demo store]">
      <![CDATA[java.lang.AssertionError: expected [nopCommerce demo store. Login] but found [nopCommerce demo store]
at Day11.DataProviderDemo.testLogin(DataProviderDemo.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="Day11.DataProviderDemo" name="testLogin" time="2.579">
    <failure type="java.lang.AssertionError" message="expected [nopCommerce demo store. Login] but found [nopCommerce demo store]">
      <![CDATA[java.lang.AssertionError: expected [nopCommerce demo store. Login] but found [nopCommerce demo store]
at Day11.DataProviderDemo.testLogin(DataProviderDemo.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
</testsuite> <!-- Test12 -->
